README.md

```diff
@@ -162,11 +193,181 @@ $ helm delete my-splunk-otel-collector
+|                   1 |              49,000 |              1024 |             1.8 |    24,000 |
+|                   2 |              20,000 |               256 |             1.3 |    20,000 |
+|                   7 |              40,000 |               256 |             2.4 |    40,000 |
+|                   5 |              58,000 |               256 |             3.2 |    54,000 |
+|                   7 |              82,000 |               256 |               3 |    52,000 |
+|                  10 |              58,000 |               256 |             3.2 |    53,000 |
+
+
+
+## Manage Splunk OpenTelemetry Collector for Kubernetes Log Ingestion by Using Annotations
+
+Manage Splunk OpenTelemetry Collector for Kubernetes Logging with these supported annotations.
+* Use `splunk.com/index` annotation on pod and/or namespace to tell which Splunk platform indexes to ingest to. Pod annotation will take precedence over namespace annotation when both are annotated.
+  ex) `kubectl annotate namespace kube-system splunk.com/index=k8s_events`
+* Use `splunk.com/sourcetype` annotation on pod to overwrite `sourcetype` field. If not set, it is dynamically generated to be `kube:container:CONTAINER_NAME` where CONTAINER_NAME is the container name of the container running in the pod.
+* Set `splunk.com/exclude` annotation to true on pod and/or namespace to exclude its logs from ingested to your Splunk platform deployment.
+* Set `splunk.com/include` annotation to true on pod and `containerLogs.useSplunkIncludeAnnotation` flag to `true` to include its logs from ingested to your Splunk platform deployment.
+  All other logs will be ignored. You cant use this feature with the above mentioned exclude feature. You can only use either the include feature or the exclude feature.
+
+## Search for Splunk OpenTelemetry Collector for Kubernetes metadata in Splunk
+Splunk OpenTelemetry Collector for Kubernetes sends events to Splunk which can contain extra meta-data attached to each event. Metadata values such as "k8s.pod.name", "k8s.pod.uid", "k8s.deployment.name","k8s.cluster.name", "k8s.namespace.name", "k8s.node.name", "k8s.pod.start_time", "container_name", "run_id" and "stream" will appear as fields when viewing the event data inside Splunk.
+There are two solutions for running searches in Splunk on meta-data.
+
+* Modify search to use`fieldname::value` instead of `fieldname=value`.
+* Configure `fields.conf` on your downstream Splunk system to have your meta-data fields available to be searched using `fieldname=value`. Example: [fields.conf.example](https://github.com/splunk/sck-otel/blob/main/example/fields.conf.example)
+
+For more information on index time field extraction please view this [guide](https://docs.splunk.com/Documentation/Splunk/latest/Data/Configureindex-timefieldextraction#Where_to_put_the_configuration_changes_in_a_distributed_environment).
+
+## Advanced Configurations for Splunk OpenTelemetry Collector for Kubernetes
```

Comment:
remove everything that is added below.

Reply:
removed - i think. take a look in case i removed something wrong